### DISCUSSION TITLE
Copilot solved my math problems

### DISCUSSION THREAD

**Discussion by EduApps-CDG** [community/community#40487] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot is so awesome that it can solve my math problems:
![image](https://user-images.githubusercontent.com/32627482/204928440-7ca5b5f7-6481-4959-ab42-b61c6c706cd7.png)

**Comment by Akash1134** [comment]
Hi @EduApps-CDG , That's great! Thanks for sharing it with us. 😄  Just a recommendation, you can also check out [8 things you didn’t know you could do with GitHub Copilot](https://github.blog/2022-09-14-8-things-you-didnt-know-you-could-do-with-github-copilot/).
